src/hdf5/osw_2ioda.py

```diff
@@ -204,8 +204,11 @@ def get_reference_time(afile, osw_source):
         dat_ref = afile['time'].attrs['units'][-19:].decode('UTF-8')
         dat_ref = datetime.strptime(dat_ref, '%Y-%m-%d %H:%M:%S').replace(tzinfo=timezone.utc).timestamp()
     elif osw_source in ('Spire'):
-        dat_ref = afile['sample_time'].attrs['units'][-29:-3].decode('UTF-8')
```

Comment:
@smaticka this was what was causing the crash on certain files... the fractional seconds can vary in precision

so just changed to a generic split and then dropped the fractional seconds which for this data type is not critical